src/reducer/admin/references/possibleValues.js

```diff
@@ -9,6 +12,11 @@ export default (previousState = initialState, { type, payload, meta }) => {
                 ...previousState,
                 [meta.relatedTo]: payload.data.map(record => record.id),
             };
+        case CRUD_GET_MATCHING_FAILURE:
+            return {
+                ...previousState,
+                [meta.relatedTo]: { error: payload.error },
```

Comment:
I'm not very fan of having either an object or an array for this state value... Does anyone have a better solution in mind?

Reply:
Add another property ?

```js
[`${meta.relatedTo}`_error]
```

Reply:
We already use this pattern otherwise, not a big deal IMO

Reply:
Okay, I'll leave it like that.